### ISSUE TITLE
multi-finger de-jittering needed!

### ISSUE THREAD

**Issue by Hevinxue** [imbushuo/mac-precision-touchpad#519]
the current 4187 release is good on single-finger operation, but when scrolling with two or multi fingers (which is quite common when viewing pages or zooming in on something), the jitter issue still happens.  

the de-jittering and anti-miscontact part is the last missing piece of this masterpiece @imbushuo

**Comment by micleeso** [comment]
I agree with the de-jittering anti-miscontact part. two-finger scroll etc works. Otherwise, this driver is working well.  rev 3979 windows 10
